src/lib.rs

```diff
@@ -13,137 +13,12 @@
     clippy::undocumented_unsafe_blocks
 )]
 
-use core::{fmt, ops::DerefMut};
+use core::{convert::Infallible, fmt, ops::DerefMut};
 
 pub mod block;
 pub mod utils;
 mod word;
 
-/// Implementation-level interface for RNGs
-///
-/// This trait encapsulates the low-level functionality common to all
-/// generators, and is the "back end", to be implemented by generators.
-/// End users should normally use the [`rand::Rng`] trait
```

Comment:
It's worth to keep this reference to `rand::Rng` in the trait docs.

Reply:
I didn't even make a "low effort" on the docs yet (until I have a better idea of whether we're doing this); some stuff will need un-deleting.